Sources/ComposableArchitecture/Store.swift

```diff
@@ -170,11 +174,24 @@ public final class Store<State, Action> {
         return .none
       }
     )
-    self.$state.producer
-      .startWithValues { [weak localStore] newValue in localStore?.state = toLocalState(newValue) }
+    let observer = Signal<State, Never>.Observer(
+      value: { [weak localStore] state in
+        localStore?.state = toLocalState(state)
+      },
+      failed: .none,
+      completed: { [weak localStore] in
+        localStore?.parentDisposable?.dispose()
+        localStore?.parentDisposable = nil
+      },
+      interrupted: { [weak localStore] in
+        localStore?.parentDisposable?.dispose()
+        localStore?.parentDisposable = nil
+      }
+    )
+    localStore.parentDisposable = self.$state.producer.start(observer)
```

Comment:
Same issue here. Why not just `localStore.parentDisposable = self.$state.producer.startWithValues { ... }` ?

Reply:
Done ✅